src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/strategy/StrategyContainerTest.java

```diff
@@ -0,0 +1,47 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd.strategy;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.servicebus.model.Classification;
+import uk.gov.hmcts.reform.bulkscan.orchestrator.services.servicebus.model.Envelope;
+
+import java.io.IOException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.SampleData.envelopeJson;
+import static uk.gov.hmcts.reform.bulkscan.orchestrator.SampleData.objectMapper;
+
+@RunWith(MockitoJUnitRunner.class)
+public class StrategyContainerTest {
+
```

Comment:
I have the same issue with this, it took me longer than it should have to understand what it does after reading the class name.